gunicorn/metadata.csv

```diff
@@ -1,6 +1,6 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
 gunicorn.requests,rate,10,request,second,The rate of requests received.,0,gunicorn,requests
-gunicorn.workers,gauge,10,worker,,Number of workers managed by the arbiter.,0,gunicorn,workers
+gunicorn.workers,gauge,10,worker,,The number of workers by state (idle or working).,0,gunicorn,workers
```

Comment:
Nice. To be extra clear, we might mention it's _tagged_ by state i.e.

"The number of workers, tagged by state (e.g. state:idle, state:working)"